independent-projects/qute/core/src/main/java/io/quarkus/qute/ValueResolver.java

```diff
@@ -47,6 +50,14 @@ default ValueResolver getCachedResolver(EvalContext context) {
         return this;
     }
 
+    default Set<String> getSupportedProperties() {
```

Comment:
What exactly are these methods used for?

We have plenty of value resolvers (even generated) at runtime. Is it necessary for `qute-debug` to work properly?